cs-amazon/src/main/java/io/cloudslang/content/amazon/entities/constants/Descriptions.java

```diff
@@ -124,4 +124,17 @@
         public static final String SUCCESS_DESC = "The action ended successfully.";
         public static final String FAILURE_DESC = "An error has occurred while trying to get details about the product.";
     }
+
+    public static class UnprovisionProductAction
+    {
+        public static final String UNPROVISION_PRODUCT_DESCRIPTION = "Terminates the specified provisioned product.";
+        public static final String PROVISIONED_PRODUCT_ID_DESC = "The identifier of the provisioned product. You cannot specify both ProvisionedProductName and ProvisionedProductId inputs.";
```

Comment:
It will be good to provide examples of the format of the data that user should provide